client/reader/list-stream/index.jsx

```diff
@@ -65,7 +66,10 @@ class ListStream extends Component {
 		}
 
 		if ( list ) {
-			this.title = list.title;
+			// Show author name in parentheses if the list is owned by someone other than the current user
+			const isOwnedByCurrentUser =
+				this.props.currentUser && list.owner === this.props.currentUser.username;
+			this.title = isOwnedByCurrentUser ? list.title : `${ list.title } (${ list.owner })`;
```

Comment:
Should this be translatable, for RTL languages?

Reply:
The list title and owner name are input by the owning user, so they are in whatever language the owner created them in. I don't think we have control to translate anything here, like we do with more predictable strings we can control.

We use `AutoDirection` to help formatting for when the RTL/LTR mismatches that may occur when the users interface is in one directionality and the other user's content being displayed is in the other.

I think the only thing we can really predict enough to translate would be "Recommended Blogs", but we should probably be doing that on the backend responses that fill the list title data (based on the locale in the request), otherwise we will be hardcoding band-aid checks all over the front-end for it.

Reply:
> I think the only thing we can really predict enough to translate would be "Recommended Blogs",

Yes, I agree. That's something we tracked in CML-692.